numpy/core/src/multiarray/scalartypes.c.src

```diff
@@ -4226,6 +4256,17 @@ initialize_numeric_types(void)
     PyDatetimeArrType_Type.tp_@name@ = datetimetype_@name@;
     PyTimedeltaArrType_Type.tp_@name@ = timedeltatype_@name@;
 
+
+    /**end repeat**/
+
+    /**begin repeat
+     * #Type = Bool, Byte, UByte, Short, UShort, Int, UInt, Long,
+     *         ULong, LongLong, ULongLong#
+     */
+
+    Py@Type@ArrType_Type.tp_str = genint_type_str;
+    Py@Type@ArrType_Type.tp_repr = genint_type_str;
```

Comment:
I find it cleaner as a separate repeat, since it is only 2 lines. This way we avoid nested repeats and the need for `repeat1`, `repeat2`.